engine/glfw/include/GL/glfw.h

```diff
@@ -571,6 +571,7 @@ GLFWAPI void glfwSetViewType(int view_type);
 GLFWAPI void glfwSetWindowBackgroundColor(unsigned int color);
 GLFWAPI float glfwGetDisplayScaleFactor();
 GLFWAPI int glfwGetMouseLocked();
+GLFWAPI int glfwGetJoystickGenericDeviceId( int joy, char** generic_device_id );
```

Comment:
I generic device id a good name?

For HTML5 this will always return "Standard Gamepad" but perhaps in the future for other platforms there could be "Standard 8 Button 2 Stick Gamepad", "Standard 16 Button 2 Stick Gamepad" etc to describe classes of gamepads?

Reply:
Removed generic device id concept